.gitignore

```diff
@@ -37,6 +37,9 @@ _build
 *.xml
 *.iml
 
+# Temp vim .swp s
+*.swp
```

Comment:
Pro tip: this should go in your global ignore file instead. Defaults to: $HOME/.config/git/ignore

This is not blocking for merge though.